examples/mnist/scripts/add_dataset_objective.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2018 Owkin, inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import os
+import zipfile
+from contextlib import contextmanager
+from types import SimpleNamespace
+
+from tqdm import tqdm
+
+import substra
+
+default_stream_handler = logging.StreamHandler()
+substra_logger = logging.getLogger('substra')
+substra_logger.addHandler(default_stream_handler)
+
+
+@contextmanager
+def progress_bar(length):
+    """Provide progress bar for for loops"""
+    pg = tqdm(total=length)
+    progress_handler = logging.StreamHandler(SimpleNamespace(write=lambda x: pg.write(x, end='')))
+    substra_logger.removeHandler(default_stream_handler)
+    substra_logger.addHandler(progress_handler)
+    try:
+        yield pg
+    finally:
+        pg.close()
+        substra_logger.removeHandler(progress_handler)
+        substra_logger.addHandler(default_stream_handler)
+
+
+current_directory = os.path.dirname(__file__)
+assets_directory = os.path.join(current_directory, '../assets')
+
+client = substra.Client(profile_name="node-1")
+client.login()
+
+DATASET = {
+    'name': 'Mnist',
+    'type': 'npy',
+    'data_opener': os.path.join(assets_directory, 'dataset/opener.py'),
+    'description': os.path.join(assets_directory, 'dataset/description.md'),
```

Comment:
I am getting a permissions issues (`{'permissions': ['This field may not be null.']}`), can you please update as it was done here: https://github.com/SubstraFoundation/substra/pull/168